src/Illuminate/Auth/DatabaseUserProvider.php

```diff
@@ -1,146 +1,151 @@
-	 */
-	protected $hasher;
-
-	/**
-	 * The table containing the users.
-	 *
-	 * @var string
-	 */
-	protected $table;
-
-	/**
-	 * Create a new database user provider.
-	 *
-	 * @param  \Illuminate\Database\ConnectionInterface  $conn
-	 * @param  \Illuminate\Contracts\Hashing\Hasher  $hasher
-	 * @param  string  $table
-	 * @return void
-	 */
-	public function __construct(ConnectionInterface $conn, HasherContract $hasher, $table)
-	{
-		$this->conn = $conn;
-		$this->table = $table;
-		$this->hasher = $hasher;
-	}
-
-	/**
-	 * Retrieve a user by their unique identifier.
-	 *
-	 * @param  mixed  $identifier
-	 * @return \Illuminate\Contracts\Auth\Authenticatable|null
-	 */
-	public function retrieveById($identifier)
-	{
-		$user = $this->conn->table($this->table)->find($identifier);
-
-		return $this->getGenericUser($user);
-	}
-
-	/**
-	 * Retrieve a user by by their unique identifier and "remember me" token.
-	 *
-	 * @param  mixed   $identifier
-	 * @param  string  $token
-	 * @return \Illuminate\Contracts\Auth\Authenticatable|null
-	 */
-	public function retrieveByToken($identifier, $token)
-	{
-		$user = $this->conn->table($this->table)
+class DatabaseUserProvider implements UserProviderInterface
+{
+    /**
+     * The active database connection.
+     *
+     * @var \Illuminate\Database\ConnectionInterface
+     */
+    protected $conn;
+
+    /**
+     * The hasher implementation.
+     *
+     * @var \Illuminate\Contracts\Hashing\Hasher
+     */
+    protected $hasher;
+
+    /**
+     * The table containing the users.
+     *
+     * @var string
+     */
+    protected $table;
+
+    /**
+     * Create a new database user provider.
+     *
+     * @param \Illuminate\Database\ConnectionInterface $conn
+     * @param \Illuminate\Contracts\Hashing\Hasher     $hasher
+     * @param string                                   $table
+     *
+     * @return void
+     */
+    public function __construct(ConnectionInterface $conn, HasherContract $hasher, $table)
+    {
+        $this->conn = $conn;
+        $this->table = $table;
+        $this->hasher = $hasher;
+    }
+
+    /**
+     * Retrieve a user by their unique identifier.
+     *
+     * @param mixed $identifier
+     *
+     * @return \Illuminate\Contracts\Auth\Authenticatable|null
+     */
+    public function retrieveById($identifier)
+    {
+        $user = $this->conn->table($this->table)->find($identifier);
+
+        return $this->getGenericUser($user);
+    }
+
+    /**
+     * Retrieve a user by by their unique identifier and "remember me" token.
+     *
+     * @param mixed  $identifier
+     * @param string $token
+     *
+     * @return \Illuminate\Contracts\Auth\Authenticatable|null
+     */
+    public function retrieveByToken($identifier, $token)
+    {
+        $user = $this->conn->table($this->table)
                                 ->where('id', $identifier)
                                 ->where('remember_token', $token)
                                 ->first();
```

Comment:
I have no automated way to fix such cases. Also, I don't think PSR-2 says anything about this.